app/src/main/java/org/mozilla/fenix/settings/AccountAuthErrorPreference.kt

```diff
@@ -11,23 +11,29 @@ import android.widget.TextView
 import androidx.preference.Preference
 import androidx.preference.PreferenceViewHolder
 import org.mozilla.fenix.R
+import kotlin.properties.Delegates
 
 class AccountAuthErrorPreference @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     attributeSetId: Int = android.R.attr.preferenceStyle
 ) : Preference(context, attrs, attributeSetId) {
-    var email: String? = null
+    private var emailView: TextView? = null
+    var email: String? by Delegates.observable<String?>(null) { d, old, new -> updateEmailView(new) }
```

Comment:
It's best practice to use underscore _ for the variable names you aren't using in a lambda. This may show up in warnings or lint checks.

Reply:
Good catch!